platforms/nuttx/src/px4/stm/stm32_common/include/px4_arch/hw_description.h

```diff
@@ -115,6 +115,7 @@ static inline constexpr uint32_t timerBaseRegister(Timer::Timer timer)
 
 	case Timer::Timer4: return STM32_TIM4_BASE;
 #ifdef STM32_TIM5_BASE
+
```

Comment:
Please revert this non-change.

Reply:
Tried reverting, looks like this was an astyle format change. It causes a pre-commit error if I revert and try to commit.